server/routes/induction/update/inPrisonWorkUpdateController.test.ts

```diff
@@ -124,7 +122,7 @@ describe('inPrisonWorkUpdateController', () => {
       expect(req.session.inductionDto).toEqual(inductionDto)
     })
 
-    it('should get the In Prison Work view given short question set journey', async () => {
+    it('should get the In Prison Work view given there is an updateInductionQuestionSet on the session', async () => {
```

Comment:
Renamed quite a few of the tests like this - the scenario is not really about whether there is a short or long question set journey; the important factor in the tests is actually if we are doing an update of the question set, as identified by `updateInductionQuestionSet` being on the session